Vagrantfile

```diff
@@ -164,10 +164,10 @@ Vagrant.configure(2) do |config|  # the literal "2" is required.
         master_config.vm.synced_folder s[0], "/#{s[1]}", :owner => "vagrant", :group => "staff", :mount_options => ["umask=0002"]
       end
     end
-    if vagrant_command == "ssh"
-      master_config.ssh.username = my_linux_user  # if you type "vagrant ssh", use this username
-      master_config.ssh.private_key_path = Dir.home() + "/.ssh/id_rsa"
-    end
+    #if vagrant_command == "ssh"
+    #  master_config.ssh.username = my_linux_user  # if you type "vagrant ssh", use this username
+    #  master_config.ssh.private_key_path = Dir.home() + "/.ssh/id_rsa"
+    #end
```

Comment:
Why's this commented-out? Should it be deleted?

Reply:
I left this in on purpose, as an example of what someone might want to do.  I thought this was really cool when I put it in, but it has ended up being a foot gun too often, so I commented it and will use a normal "ssh -A bevymaster" when I want to log in using my own username, and "vagrant ssh bevymaster" will log in using the default "vagrant" username.